cdxev/amend/replace_license_name_with_id.py

```diff
@@ -70,4 +86,47 @@ def replace_license_name_with_id(component: dict, license_name_id_list: list) ->
         if id_found:
             current_license["id"] = id_found
             current_license.pop("name")
+        elif path_to_license_folder:
+            license_text = get_license_text_from_folder(
+                current_license.get("name", ""), path_to_license_folder
+            )
+            if license_text == "":
+                component_id = ComponentIdentity.create(component, allow_unsafe=True)
+                logger.warning(
+                    LogMessage(
+                        "License text not found",
+                        (
+                            f"No text for the license ({current_license.get('name', '')}), "
+                            f"in component ({component_id}), was found. "
+                            "An empty string was added as text."
+                        ),
+                    )
+                )
+            current_license["text"] = license_text
```

Comment:
There should be a warning that text is overwritten. It's not sufficient to only state it in the documentation.